README.md

```diff
@@ -33,7 +33,9 @@ The build tools also include a CLI, which can be used by installing the tools gl
 | ----------------------- | ----------- | -------------------------------------------------------------------------- |
 | `destination` (*)       | `string`    | Target location for the Material build.                                    |
 | `modules`               | `string[]`  | Modules that should be part of the build.<br/> All modules will be built if nothing is specified.                                                                                                           |
-| `version`               | `string`    | Version of AngularJS Material.<br/> If set to `local`, it will take the local installed AngularJS Material version from the node modules. <br/> If set to `latest`, the latest version will be downloaded.                                                           |
+| `excludeModules`               | `string[]`  | Modules that should be excluded from the build.<br/> Use to exclude e.g. core modules when building a bundle that should get lazy-loaded.                                                                                                           |
+| `excludeMainModule`               | `boolean`  | Set to `true` to exclude the code for the Angular Material main module. Use when building a bundle that will get lazy-loaded and extend an already existing main module.                                                                 |
+| `version`               | `string`    | Version of AngularJS Material.<br/> If set to `local`, it will take the local installed Angular Material version from the node modules. <br/> If set to `latest`, the latest version will be downloaded.                                                           |
```

Comment:
Nit: if you still have a second, can you please keep this above because this more of a "prioritized" option. The rest isn't sorted either 😄